src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/DynatraceMetricsExporterTests.cs

```diff
@@ -387,68 +418,66 @@ public async Task Export_MultipleMetricStreams_ShouldExportMultipleLines()
 				.AddReader(metricReader)
 				.Build();
 
-			var counterA = meter.CreateCounter<int>("counterA");
-			counterA.Add(10, _attributes);
+			var counter = meter.CreateCounter<int>("counter");
 
-			var counterB = meter.CreateCounter<int>("counterB");
-			counterB.Add(20, _attributes);
+			for (var i = 10; i <= 30; i += 10)
+			{
+				counter.Add(i, _attributes);
 
-			// Act - Reader will call our exporter
-			metricReader.Collect();
+				metricReader.Collect();
 
-			// Assert
-			var exportedMetrics = metricReader.GetExportedMetrics();
-			var pointA = MetricTest.FromMetricPoints(exportedMetrics.First().GetMetricPoints()).First();
-			var pointB = MetricTest.FromMetricPoints(exportedMetrics.Last().GetMetricPoints()).First();
+				// Assert
+				var exportedMetrics = metricReader.GetExportedMetrics();
+				var point = MetricTest.FromMetricPoints(exportedMetrics.First().GetMetricPoints()).First();
 
-			var expected = @$"counterA,attr1=v1,attr2=v2,dt.metrics.source=opentelemetry count,delta=10 {pointA.TimeStamp}
-counterB,attr1=v1,attr2=v2,dt.metrics.source=opentelemetry count,delta=20 {pointB.TimeStamp}";
+				var expected = $"counter,attr1=v1,attr2=v2,dt.metrics.source=opentelemetry count,delta={i} {point.TimeStamp}";
+				var actualMetricString = await actualRequestMessage.Content!.ReadAsStringAsync();
+				Assert.Equal(expected, actualMetricString);
 
-			var actualMetricString = await actualRequestMessage.Content!.ReadAsStringAsync();
-			Assert.Equal(expected, actualMetricString);
+				AssertExportRequest(actualRequestMessage);
+			}
 
 			mockMessageHandler.Protected().Verify(
 				"SendAsync",
-				Times.Exactly(1),
+				Times.Exactly(3),
```

Comment:
Why does it export three times? 1 for Delta, 1 for Cumulative, right? Or is cumulative exported twice?

Reply:
Because the for above goes 3 times, in order to check if delta export is working. 10 > 20 > 30. It should export deltas as the same values. `count,delta={i}`